### PR TITLE
Test CI

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=efc409ba98d2)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=720f46597770)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4b192caad25a)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5b6cbbdc3da9)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=63c3a87384fb)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a67e8d65b51d)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=05c0ba3034d4)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=060e2e67627e)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f3b18c925adf)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4330be454c77)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9b36715ae176)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5ddc1994e145)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d8765e48a9f1)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=38cd75ce1daf)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=566b686e4c95)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7f1b5abda00a)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d966f6a749eb)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5d125cfdac2a)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d1c3d86414c3)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f879ae97e78c)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b8be5be5cc2d)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e1335fe59f25)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5be52de9b814)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f1d9291a3ac8)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a9222a713c05)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0e9f5721b479)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e4733045118d)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=56b3d3db97d7)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2eeacfabbcb5)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f833fb0408e7)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=95b2777ca41a)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Test CI
+
 # Gaia [![Build Status](https://travis-ci.org/mozilla-b2g/gaia.svg)](https://travis-ci.org/mozilla-b2g/gaia)
```
Comment (ranglaly) [inline README.md:3]: (fire fox )send data to kai os (jiophone) apps


ajgmgptj5.6gpt78pg1 '&4j

**Review by ranglaly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**